wiki/Play_style/Alternating/fr.md

```diff
@@ -0,0 +1,27 @@
+---
+stub: true
+tags:
+  - keyboard
+  - tap
+  - alternate
+  - alt
+  - play style
+---
+
+# Alternating
+
+*A ne pas confondre avec [Double tapping](/wiki/Play_style/Double_tapping).*
+
+L'**Alternating** est un style de jeu pour le mode [osu!standard](/wiki/Game_mode/osu!) dans lequel les joueurs cliquent sur des cercles, font glisser des sliders et font tourner des spinners en alternant entre deux touches de leur clavier (`Z` et `X` par défaut) tous les deux [objets](/wiki/Hit_object) ou presque. D'où le nom. Ce style est le plus courant chez les joueurs plus expérimentés et c'est l'un des plus difficiles à adapter.
```

Comment:
Toute cette traduction est assez littérale - elle marche et est compréhensible, mais n'est pas exactement du bon français (il faut retravailler la tournure de la phrase). Remplace par ça:

L'**Alternating** est une méthode de jeu qui consiste à cliquer sur les cercles, faire glisser les sliders, ou faire tourner les spinners, en alternant entre les deux touches de leur clavier (`Z` et `X` par défaut) d'un objet à l'autre, d'où le nom. Cette méthode est plus courante chez les joueurs plus expérimentés, étant plus difficiles à apprendre.

("every other" - "chaque autre" littéralement ne veut pas dire tous les deux, mais une fois sur deux)
("adapt to" veut dire "s'adapter à", mais ce verbe ne fonctionne pas dans ce context, il faut le remplacer par un autre qui garde le même sens)